src/media/css/tutorial.styl

```diff
@@ -27,6 +27,10 @@
 
     transition: opacity 1s;
 
+    &.invisible {
+        opacity: 0;
```

Comment:
Might be smart to add a transition so it looks like the intended effect. I'd also recommend a spinner to let the user know something is loading, though in the screenshot for the bug it seems like there might already be one.